CorsixTH/Src/th_lua_anims.cpp

```diff
@@ -182,6 +182,12 @@ int l_anims_set_marker(lua_State* L) {
   return 1;
 }
 
+int l_anims_tick(lua_State* L) {
+  animation_manager* pAnims = luaT_testuserdata<animation_manager>(L);
+  pAnims->tick();
+  return 1;
```

Comment:
return 0. That's the number of return values pused to the lua stack.

Reply:
Thanks! I thought it was some kind of success code or something, heh. Fixed.